src/commands/codepush/lib/release-command-skeleton.ts

```diff
@@ -1,14 +1,29 @@
 import { AppCommand, CommandResult, ErrorCodes, failure, hasArg, help, longName, shortName, success, defaultValue } from "../../../util/commandline";
+import { CommandArgs } from "../../../util/commandline/command";
 import { AppCenterClient, models, clientRequest } from "../../../util/apis";
 import { out, prompt } from "../../../util/interaction";
+import { getUser, DefaultApp } from "../../../util/profile/index";
 import { inspect } from "util";
 import * as fs from "fs";
 import * as pfs from "../../../util/misc/promisfied-fs";
 import * as chalk from "chalk";
 import { sign, zip } from "../lib/update-contents-tasks";
 import { isBinaryOrZip } from "../lib/file-utils";
+import { environments } from "../lib/environment";
 import { isValidVersion, isValidRollout, isValidDeployment } from "../lib/validation-utils";
-import * as debug from "debug";
+import { AppCenterCodePushRelease, LegacyCodePushRelease }  from "../lib/release-strategy/index";
+
+const debug = require("debug")("mobile-center-cli:commands:codepush:release-skeleton");
```

Comment:
Please change debug string to `"appcenter-cli:commands:..."`

Reply:
fixed